pool/math_logic.gno

```diff
@@ -96,25 +105,25 @@ func DrySwap(
 		}
 
 		step.sqrtPriceNextX96 = TickMathGetSqrtRatioAtTick(step.tickNext)
-		var _sqrtRatioTarget bigint
 
+		var sqrtPriceX96Target bigint
 		if zeroForOne {
 			if step.sqrtPriceNextX96 < sqrtPriceLimitX96 {
-				_sqrtRatioTarget = sqrtPriceLimitX96
+				sqrtPriceX96Target = sqrtPriceLimitX96
 			} else {
-				_sqrtRatioTarget = step.sqrtPriceNextX96
+				sqrtPriceX96Target = step.sqrtPriceNextX96
 			}
 		} else {
 			if step.sqrtPriceNextX96 > sqrtPriceLimitX96 {
-				_sqrtRatioTarget = sqrtPriceLimitX96
+				sqrtPriceX96Target = sqrtPriceLimitX96
 			} else {
-				_sqrtRatioTarget = step.sqrtPriceNextX96
+				sqrtPriceX96Target = step.sqrtPriceNextX96
 			}
 		}
```

Comment:
```suggestion
isLower := step.sqrtPriceNextX96 < sqrtPriceLimitX96
isHiger := step.sqrtPriceNextX96 > sqrtPriceLimitX96

if (zeroForOne && isLower) || (!zeroForOne && isHigher) {
	sqrtPriceX96Target = sqrtPriceLimitX96
} else {
	sqrtPriceX96Target = step.sqrtPriceNextX96
}
```

Reply:
done in https://github.com/gnoswap-labs/gnoswap/pull/98/commits/6ca30e4a4ce1e464cb096013b090548f767c1c67